applications/GeoMechanicsApplication/custom_processes/apply_c_phi_reduction_process.cpp

```diff
@@ -64,7 +66,14 @@ void ApplyCPhiReductionProcess::ExecuteFinalize()
     KRATOS_INFO("ApplyCPhiReductionProcess") << "Final safety factor = " << 1.0 / mReductionFactor << std::endl;
 }
 
-double ApplyCPhiReductionProcess::GetAndCheckPhi(const Element::PropertiesType& rProp)
+int ApplyCPhiReductionProcess::Check()
```

Comment:
Something is strange with git on my laptop and I cannot switch to your branch. Please could you explain where Check function is called? Thank you

Reply:
The comment for this in the process base class is:
```c++
/* @brief This function is designed for being called after ExecuteInitialize ONCE
     * to verify that the input is correct.
     * /
```